networkmanager/networkmanager.go

```diff
@@ -517,6 +522,9 @@ func (manager *NetworkManager) addNetwork(
 }
 
 func (manager *NetworkManager) getIPSubnet(networkID string) (allocIPNet *net.IPNet, err error) {
+	manager.Lock()
```

Comment:
I guess `ipamSubnetwork` should have own mutex to guard `usedIPSubnetNetworks`.

Reply:
Done